Makefile

```diff
@@ -1,43 +1,52 @@
-DESTDIR ?= /
-SBINDIR = $(DESTDIR)/usr/sbin
-SYSCONFDIR = $(DESTDIR)/etc
-MANDIR = $(DESTDIR)/usr/share/man
-LOGDIR = $(DESTDIR)/var/log/recap
-CRONDIR = $(DESTDIR)/etc/cron.d
+DESTDIR       ?=
+BINPATH       ?= /sbin
+PREFIX        ?= /usr/local
+BINDIR        ?= $(PREFIX)$(BINPATH)
+MANDIR        ?= $(PREFIX)/share/man
+DOCDIR        ?= $(PREFIX)/share/doc
+SYSCONFDIR    ?= /etc
+CRONDIR       ?= $(SYSCONFDIR)/cron.d
+LOGDIR        ?= /var/log
 
 all:
 	@echo "Nothing to compile, run make install."
 
 install:
 	@echo "Installing scripts..."
-	@install -Dm0755 src/recap $(SBINDIR)/recap
-	@install -Dm0755 src/recaplog $(SBINDIR)/recaplog
-	@install -Dm0755 src/recaptool $(SBINDIR)/recaptool
+	@install -Dm0755 src/recap $(DESTDIR)$(BINDIR)/recap
+	@install -Dm0755 src/recaplog $(DESTDIR)$(BINDIR)/recaplog
+	@install -Dm0755 src/recaptool $(DESTDIR)$(BINDIR)/recaptool
 	@echo "Installing man pages..."
-	@install -Dm0644 src/recap.5 $(MANDIR)/man5/recap.5
-	@install -Dm0644 src/recap.8 $(MANDIR)/man8/recap.8
-	@install -Dm0644 src/recaplog.8 $(MANDIR)/man8/recaplog.8
+	@install -Dm0644 src/recap.5 $(DESTDIR)$(MANDIR)/man5/recap.5
+	@install -Dm0644 src/recap.8 $(DESTDIR)$(MANDIR)/man8/recap.8
+	@install -Dm0644 src/recaplog.8 $(DESTDIR)$(MANDIR)/man8/recaplog.8
+	@install -Dm0644 src/recaptool.8 $(DESTDIR)$(MANDIR)/man8/recaptool.8
 	@echo "Installing configuration..."
-	@install -Dm0644 src/recap.conf $(SYSCONFDIR)/recap
```

Comment:
alright, I've created #80 to keep the discussion of the config file, since there is no other problem with the Makefile I'll proceed to merge it.